CONTRIBUTING.md

```diff
@@ -107,10 +107,32 @@ This section shows you how to send a pull request. The main staff are always mon
 
 This section shows you how to write commit messages in Clay. Follow these guidelines to help us maintain order and make it easier to locate your changes.
 
-#### Before You Make a Commit Message
+Each commit message consists of a header, a body and a footer. The header has a special format that includes a type, a scope and a subject:
 
--   Make sure that the problem or change you are making doesn't already exists.
--   Start the commit message with the word **Fixes**, and always follow it with a reference to the issue: - Fixes #0 - bla bla - Fixes #0, #1 - bla bla
+```
+<type>(<scope>): <subject>
+```
+
+The header is mandatory and the scope of the header is optional.
+
+> Clay follows the "[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)" specification.
```

Comment:
Just link to https://www.conventionalcommits.org/ — it will redirect to the latest, which is already: https://www.conventionalcommits.org/en/v1.0.0-beta.4/